### PR TITLE
chore(metadata-sidebar): mock storybook queries for MetadataSidebarRedesign

### PR DESCRIPTION
**Pull request by wpiesiak**
### Description
Mock API requests data in `MetadataSidebarRedesign` stories

### CODE REVIEW

File: src/elements/content-sidebar/stories/MetadataSidebarRedesign.stories.tsx [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
+import { http, HttpResponse } from 'msw';
 import MetadataSidebarRedesign from '../MetadataSidebarRedesign';
 import ContentSidebar from '../ContentSidebar';
+import {
+    mockEnterpriseMetadataTemplates,
+    mockFileRequest,
+    mockMetadataInstances,
+} from './__mocks__/MetadataSidebarRedesignedMocks';
 
 const fileIdWithMetadata = global.FILE_ID;
```
Comment (karolinaru) [inline src/elements/content-sidebar/stories/MetadataSidebarRedesign.stories.tsx:13]: shouldn't we import the variable from mock data now or it doesn't matter here?

File: src/elements/content-sidebar/stories/tests/MetadataSidebarRedesign-visual.stories.tsx [hunk 2]
```diff
@@ -1,12 +1,21 @@
...
+    fileId,
+    fileIdWithoutMetadata,
+    mockEmptyMetadataInstances,
+    mockEnterpriseMetadataTemplates,
+    mockFileRequest,
+    mockFileRequestWithoutMetadata,
+    mockMetadataInstances,
+} from '../__mocks__/MetadataSidebarRedesignedMocks';
+
+const fileIdWithMetadata = fileId;
```
Comment (kajarosz) [inline src/elements/content-sidebar/stories/tests/MetadataSidebarRedesign-visual.stories.tsx:18]: Can't it be named using `fileIdWithMetadata` when it's defined?

File: src/elements/content-sidebar/stories/__mocks__/MetadataSidebarRedesignedMocks.ts [hunk 3]
```diff
@@ -0,0 +1,323 @@
...
+    url: `https://api.box.com/2.0/files/${fileId}/metadata`,
+    response: {
+        entries: [
+            {
+                $id: '776397f2-5d56-4a33-80c1-569614702899',
+                $version: 1,
+                $type: 'myTemplate-453ffc2f-bf5a-464c-a004-476a4eac20fd',
+                $parent: 'file_415542803939',
+                $typeVersion: 1,
+                $template: 'myTemplate',
```
Comment (kajarosz) [inline src/elements/content-sidebar/stories/__mocks__/MetadataSidebarRedesignedMocks.ts:73]: Shouldn't this be `templateKey`?

Comment (jankowiakdawid) [reply src/elements/content-sidebar/stories/__mocks__/MetadataSidebarRedesignedMocks.ts:73]: No, https://developer.box.com/reference/resources/metadata--full/#param-$template

File: src/elements/content-sidebar/stories/MetadataSidebarRedesign.stories.tsx (hunk 1 above)
Comment (jankowiakdawid) [reply src/elements/content-sidebar/stories/MetadataSidebarRedesign.stories.tsx:13]: I think it doesn't matter much. And if we start using this mocks for all stories, we can stop relying data coming from test account all together.

File: src/elements/content-sidebar/stories/tests/MetadataSidebarRedesign-visual.stories.tsx (hunk 2 above)
Comment (jankowiakdawid) [reply src/elements/content-sidebar/stories/tests/MetadataSidebarRedesign-visual.stories.tsx:18]: Wiola is off till Nov 13, and I don't have access to her fork, so I think we can rename it in future PRs.

**Review by jankowiakdawid — APPROVED** [review]

**Review by karolinaru — APPROVED** [review]

**Review by JChan106 — APPROVED** [review]

**Pull request merged**